libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/ExpandedManifest.tsx

```diff
@@ -33,11 +34,18 @@ const ExpandedManifest = ({ fieldName, manifestIdx }: CustomManifestComponentPro
         <GridItem span={6}>
           <OcmInputField
             name={`${fieldName}.filename`}
-            label="File name"
             isRequired
             data-testid={`filename-${manifestIdx}`}
-            helperText={
-              'File name determines order of executing manifests during the installation. For example "manifest1.yaml" will be applied before "manifest2.yaml"'
+            helperText={'Use yaml, yml or JSON file types. File size must not exceed 1MB.'}
```

Comment:
Is there a constant somewhere for the maximum manifest size? For example, are we showing a message when the size exceeeds 1MB?

It's better to reference the constant everywhere to avoid inconsistent messages.

Reply:
Yes, it's true. I've used MAX_FILE_SIZE_BYTES constant that we are using to validate file size.